server/app.js

```diff
@@ -6,6 +6,10 @@ import userRoutes from './routes/user.route';
 import swaggerUi from 'swagger-ui-express';
 import swaggerDoc from './swagger.json';
 
+import Database from './db/db';
+
+Database.createScripts();
```

Comment:
I gave you feedback about this yesterday.
You shouldn't be doing this here

Reply:
is one task but , there are configuration on packsagejson i install and some deletion of folder that , is look on staged area as many file but is one file of app only

Reply:
am finish the to do it ,

Reply:
can i merge it

Reply:
could you check it again